src/components/adminpanel/adminpanel.jsx

```diff
@@ -23,71 +22,45 @@ class AdminPanel extends React.Component {
     }
     render () {
         if (!this.props.isAdmin) return false;
-
-        if (this.state.showPanel) {
-            return (
-                <div
-                    className="visible"
-                    id="admin-panel"
-                >
+        return (
+            <div
+                className={classNames(
+                    'admin-panel', this.props.className, {
+                        hidden: !this.state.showPanel
+                    }
+                )}
+            >
+                {this.state.showPanel ? (
+                    <React.Fragment>
+                        <span
+                            className="toggle"
+                            onClick={this.handleToggleVisibility}
+                        >
+                            x
+                        </span>
+                        <div className="admin-header">
+                            <h3>Admin Panel</h3>
+                        </div>
+                        <div className="admin-content">
+                            {this.props.children}
+                        </div>
+                    </React.Fragment>
+                ) : (
                     <span
                         className="toggle"
                         onClick={this.handleToggleVisibility}
                     >
-                        x
+                        &gt;
```

Comment:
this makes for a pretty small click target. might want to use an enclosing circle or something, as a follow-on at some point.

Reply:
Sure! This is just what already existed for the admin panel we have on the homepage.